builds/fccsw-build.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+#---Create stampfile to enable our jenkins to purge old builds------------------------------
+touch $WORKSPACE/controlfile
```

Comment:
Do we need this even though we purge fccsw builds now by default?

Reply:
Yes, the `controlfile` is used by some jenkins jobs to clean the nodes up. IMHO it is a good practice to ensure we create it since this build script is meant to be run in the jenkins environment.